cosmoz-image-viewer.js

```diff
@@ -291,19 +291,23 @@
 		get imageOverlay() {
 			return imageOverlay;
 		},
+
+		get carousel() {
+			return this.$$('#carousel');
```

Comment:
Why not `this.$.carousel` ?

Reply:
In the tests, I prefer to reference "public" properties where possible rather than querySelect the carousel.